3_javascript/javascript3/week1/WeatherApp/scripts.js

```diff
@@ -0,0 +1,103 @@
+domListener();
+function domListener() {
+  document.addEventListener("DOMContentLoaded", domLoader);
+}
+
+function domLoader() {
+  const inputCity = document.getElementById("nameCity");
+  const searchButtom = document.getElementById("searchButtom");
+  const weatherInfo = document.getElementById("weatherInfo");
+
+  let nameOfCity = "copenhagen";
+
+  inputCity.addEventListener("input", function () {
+    nameOfCity = inputCity.value.toLowerCase();
+  });
+
+  searchButtom.addEventListener("click", weatherFunction);
+
+  function weatherFunction() {
+    weatherInfo.innerText = "";
+    fetch(
+      `https://api.openweathermap.org/data/2.5/weather?q=${nameOfCity}&appid=62550c6942b65218b08fefa7f3bf246e&units=metric`
+    )
+      .then((response) => response.json())
+      .then((data) => {
+        console.log(data);
+
+        nameInfo(data);
+        temperatureInfo(data);
+
+        const temperature = data.main.temp;
+        const weatherIconClass = selectWeatherIcon(temperature);
+        const weatherIcon = document.createElement("i");
+        weatherIcon.className = weatherIconClass;
+        weatherInfo.appendChild(weatherIcon);
+
+        windInfo(data);
+        cloudInfo(data);
+        dayTime(data);
```

Comment:
It might be more optimal to only pass the data that you need. 
Since you separated the functions so nicely, you could also make sure you don't throw all the data into them. 
But this is also maybe a nitpick :D